src/myloader.c

```diff
@@ -399,7 +404,7 @@ int main(int argc, char *argv[]) {
 
   struct thread_data t;
   t.thread_id = 0;
-  t.conf = &conf;
+  t.conf = &conf; // TODO: if conf is singleton it must be accessed as global variable
```

Comment:
On myloader yes! I'm agree with you on this. However, on mydumper it is not.

Reply:
This is myloader code. Ok for mydumper.